### ISSUE TITLE
South African national aerial imagery 2014-2019 0.25cm RGB (Zander Venter)

### ISSUE THREAD

**Issue by zanderVenter** [samapriya/awesome-gee-community-datasets#11]
# South African national aerial imagery 2014-2019 0.25cm RGB

National Geo-spatial Information (NGI) is the government component responsible for aerial photography and has an archive of aerial photographs. This is a full-coverage mosaic of orthophotos at 25cm resolution in red-green-blue . The data includes imagery from aerial missions between 2014 and 2019. ZSV consulting is a registered data vendor with NGI and has ingested the imagery into Google Earth Engine to facilitate access for the research community. For more information please consult the NGI website here: http://www.ngi.gov.za/index.php/what-we-do/aerial-photography-and-imagery

The data products were obtained from NGI in June 2019 and have not been updated since then. The data products have not been modified in any way and the original metadata is provided with each file. We acknowledge that State copyright exists in the NGI product and in derivatives of the NGI product in accordance with the State Copyright Act, 1978 (Act 98 of 1978). We acknowledge that NGI is the custodian of the original datasets/products. 

Researchers who use this data in their research for publication will be required to seek copyright permission from the State. They can make application to Michelle Denner (michelle.denner@dalrrd.gov.za) with the details of their publication. They will be issued with a copyright authorization number from the Government Printer. 

![SAorthophoto](https://user-images.githubusercontent.com/34233488/124358240-b133f000-dc1f-11eb-83c1-57ab991a82ec.gif)

#### Earth Engine Snippet
```js
var saOrtho = ee.ImageCollection('projects/nina/SouthAfrica/Orthophoto_NGI_2018');
```

Sample Code: https://code.earthengine.google.com/ae9516232e162b9422fa1ac1664c3846

#### Usage notes and licensing
We acknowledge that State copyright exists in the NGI product and in derivatives of the NGI product in accordance with the State Copyright Act, 1978 (Act 98 of 1978). Please contact Michelle Denner (michelle.denner@dalrrd.gov.za) for copyright authorization number before publishing online.

Curated by: Zander Venter

Keywords: aerial, orthophoto, imagery, south africa, national, land cover, high resolution

Last updated: 2021-07-03

**Comment by samapriya** [comment]
Unfortunately the copyright is restrictive and does not allow for redistribution without permission . So unfortunately this cannot be included in the public catalog . I am open to thoughts and discussion on this

**Comment by zanderVenter** [comment]
Yes I worried about that. If the public catalog is restricted to CC BY 4 licenses then I understand it does not qualify. Are you open to changing the scope of the catalog to include datasets like this one? The onus is on the user to seek correct permission before publishing with it. I understand if you are not keen on that. It is just such a great resource for the research community that it is a pity not to share it.

**Comment by samapriya** [comment]
The catalog isn't restricted to CC by 4 licensing only but it definitely respects redistribution rights and licensing for all datasets. In this case you may not have the right to redistribute the data/image collection to others even if you ingested it and I agree it is a great resource/dataset. I had similar issues with few other datasets where access was open but redistribution was not allowed. Hence the problem. So premise of the data has to atleast allow for you or anyone to reshare the dataset before anything else. Hope that explains why this dataset is problematic. Did you obtain permission to reshare publicly ?

**Comment by zanderVenter** [comment]
Thanks for explaining. That makes complete sense. I guess I didn't think of the redistribution rights in that way. I realize now that sadly the premise of this data is that only registered data vendors (like myself) are allowed to redistribute the data. But as far as I understand the "legaleeze" in the MoU I signed, it is not allowed for anyone else to redistribute the data themselves without first signing an MoU. I got permission to reshare the data publicly, but I don't think it is allowed for others to redistribute the data themselves. I'm guessing that is a problem for the community dataset catalog?? Sorry for the confusion...this legal stuff is new for me.

**Comment by samapriya** [comment]
Hi @zanderVenter 
I have limited legal knowledge too . You can always reach out and let them know that you would like to redistribute via a community catalog but you or I or the catalog would not be responsible for anyone not emailing and getting permission.  I think unfortunately even though you have rights to redistribution it may not extend to me and the catalog.

 But let me know if you suggest this to the owners of the dataset about putting the data in the catalog and maybe they might have a middle ground for us to work with. Happy to work with you on this but it's just stuck because of the nature and language of redistribution and use . Thank you so much for proposing this dataset and I really hope you can share more where there might be more permissive licenses :)

**Comment by zanderVenter** [comment]
Thanks. Good idea. I have emailed the data owners and asked about this. I have used the South African national land cover dataset as an example. Hopefully they will grant permission! Will keep you updated.

**Comment by samapriya** [comment]
Hey @zanderVenter 
thought I would touch base with you. Any response from the data owners yet ? Hoping they agree to make the resource available this is indeed a wonderful dataset

**Comment by zanderVenter** [comment]
Thanks for checking in. They are still reviewing my request and have not given me feedback yet. I also really hope they agree to this. Will keep you posted!

**Comment by samapriya** [comment]
Hey @zanderVenter 
Closing this issue feel free to reopen it there are any changes
